modules/ossm-rn-new-features.adoc

```diff
@@ -103,9 +125,31 @@ Additionally, the SMMR must also be configured for cluster-wide deployment. This
 ----
 <1> Adds all namespaces to the mesh, including any namespaces you subsequently create. The following namespaces are not part of the mesh: kube, openshift, kube-* and openshift-*.
 
+== New features {SMProductName} version 2.2.5
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.9 or later.
+
+=== Component versions included in {SMProductName} version 2.2.5
+
+|===
+|Component |Version
+
+|Istio
+|1.12.9
+
+|Envoy Proxy
+|1.20.8
+
+|Jaeger
+|1.39
+
+|Kiali
+|1.48.5
```

Comment:
Kiali version for 2.2 CP is  **1.48.3**

Please, change this to Kiali **1.48.3**

![kaili_version](https://user-images.githubusercontent.com/55099523/215178908-e7f615c5-54cf-4ca7-b642-0711e37089e3.png)

Reply:
Thanks @pbajjuri20 updating now!